### PR TITLE
Refactor page objects for uitests

### PR DESCRIPTION
**Pull request by palaparthi-ccpo**
Removed the duplicated code in Application Page objects file  and refactored some of the functions names to match with UI elements so that it will not create a confusion  with other function names. Also noticed one of the test case name is defined as “14.Revoke  Application member Invite” but test steps were Covering Portfolio revoke which is different, and it was a duplicate test case so renamed that one  to a new test case name and updated the steps to cover the missing functionality.
Moved all the page objects to their corresponding files to match with UI .because of this changes, some of the test cases are updated
Added executor script to Toggle button in settings >Portfolio view Application to resolve the issue in IE browser click.
Ticket: AT-6244

### CODE REVIEW

File: uitests/framework/page_objects/application_page.py [hunk 1]
```diff
@@ -26,26 +28,19 @@ class CreateApplicationPages:
...
+    btn_toggle_menu_css = ".atat-table button"
+    btn_role_perm = "button .toggle-menu__toggle a:nth-child(1)"
     btn_save_revoke_css = "input[type='submit']"
     box_edit_app = "div:nth-child(2) > div.member-form > div > div:nth-child(1) > div > fieldset > legend > label"
-    box_edit_fund = "div:nth-child(2) > div.member-form > div > div:nth-child(2) > div > fieldset > legend > label"
-    box_edit_port = "div:nth-child(2) > div.member-form > div > div:nth-child(3) > div > fieldset > legend > label"
-    box_edit_fund_remove = ".portfolio-perms > div:nth-of-type(2) > .usa-input.input__inline-fields.checked > fieldset.usa-input__choices > legend > label"
+    btn_resend_invite_b = "button .toggle-menu__toggle a:nth-child(2)"
+    btn_revoke_invite_css = "button .toggle-menu__toggle a:nth-child(3)"
+    btn_resend_invite_confirm_b = "/html/body/div/div[3]/div[2]/div/div/div[3]/div/div[1]/div[2]/div/div/div/div/form/div[2]/input"
```
Comment (kylelaker-ccpo) [inline uitests/framework/page_objects/application_page.py:37]: This feels really fragile and likely to break if any part of the page structure changes. Is there any better class/ID/selector for the button that we could use instead?

Comment (kylelaker-ccpo) [reply uitests/framework/page_objects/application_page.py:37]: For what it's worth, I _really_ like the selectors later on based on text.

File: uitests/framework/test_cases/test_resend_app_mem_invite.py [hunk 2]
```diff
@@ -49,6 +52,12 @@ def test_resend_app_mem_invite(self, setup):
         # Generator to create unique Application name
         self.appName = "App Name" + random_no_generator()
 
+        # Generator to create unique email address
+        self.email = random_generator() + "@gmail.com"
```
Comment (jasonburkert-ccpo) [inline uitests/framework/test_cases/test_resend_app_mem_invite.py:56]: example.com

File: uitests/framework/test_cases/test_revoke_env.py [hunk 3]
```diff
@@ -44,6 +47,9 @@ def test_revoke_environment(self, setup):
         # Generator to create unique email address
         self.email = random_generator() + "@gmail.com"
```
Comment (jasonburkert-ccpo) [inline uitests/framework/test_cases/test_revoke_env.py:48]: example.com

Why not make a re-usable function random_email_generator() and make this a part of it?

File: uitests/framework/test_cases/test_revoke_port_mem_invite.py [hunk 4]
```diff
@@ -30,11 +33,17 @@ def test_revoke_port_mem_invite(self, setup):
...
         )
+        # Initializing Page Objects
         self.cm = PageObjectMethods(self.driver)
         self.port = AddNewPortfolioPages(self.driver)
         self.app = CreateApplicationPages(self.driver)
         self.set = SettingsPages(self.driver)
+        # Generator to create unique Portfolio name
         self.pName = "Test Portfolio" + random_generator()
+        # Generator to create unique email address
+        self.email = random_generator() + "@gmail.com"
```
Comment (jasonburkert-ccpo) [inline uitests/framework/test_cases/test_revoke_port_mem_invite.py:44]: example.com

File: uitests/framework/page_objects/application_page.py (hunk 1 above)
Comment (jasonburkert-ccpo) [reply uitests/framework/page_objects/application_page.py:37]: Agree. Value of `btn_resend_invite_confirm_b` looks brittle.  Consider improving in the future.

Comment (palaparthi-ccpo) [reply uitests/framework/page_objects/application_page.py:37]: Agree with Kyle and Jason . Ed is the one who added these selectors and he will make changes later.

File: uitests/framework/test_cases/test_app_team_revoke_invite.py [hunk 5]
```diff
@@ -39,52 +43,71 @@ def test_revoke_app_mem_invite(self, setup):
         # Generator to create random Test Portfolio name
         self.pName = "Test Portfolio" + random_generator()
 
+        # Generator to create unique Application name
+        self.appName = "App Name" + random_no_generator()
+
+        # Generator to create unique email address
+        self.email = random_generator() + "@gmail.com"
```
[the remark this answers is not in the document]
Comment (palaparthi-ccpo) [reply uitests/framework/test_cases/test_app_team_revoke_invite.py:50]: Thanks for the feedback. I'll implement as per your suggestions

**Comment by jasonburkert-ccpo** [comment]
Discussed the gmail.com/example.com swap with @palaparthi-ccpo 
We'll address this later.

**Pull request merged**